.github/CODEOWNERS

```diff
@@ -178,7 +178,7 @@
 /sdk/search/                                         @alzimmermsft @jairmyree
 
 # PRLabel: %Service Bus
-/sdk/servicebus/                                     @anuchandy @ki1729 @conniey @liukun-msft @ZejiaJiang
+/sdk/servicebus/                                     @anuchandy @conniey @lmolkova
```

Comment:
Do we also need to tweak Event Hubs on L127?  I believe some of those folks may be focused elsewhere these days...